javascript-methods/main.js

```diff
@@ -0,0 +1,67 @@
+var one = 30;
+var two = 40;
+var three = 50;
+
+var maximumValue = Math.max(one,two,three);
+
+console.log('Value of maximumValue is: ',maximumValue);
+
+var heroes = ['superman','batman','spiderman','groot'];
+
+var randomNumber = Math.random();
+
+var randomNumber = randomNumber * heroes.length;
+
+var randomIndex = Math.floor(randomNumber);
+
+console.log('Value of random index: ', randomIndex);
+
+var randomHero = heroes[randomIndex];
+
+console.log('Value of randomHero: ',randomHero);
+
+
+var library = [
+  book1 = {
+    title:'In Search of Lost Time',
+    author: 'Marcel Proust',
+  } ,
+  book2 = {
+    title: 'Ulysses',
+    author: 'James Joyce',
+  },
+  book3 = {
+    title: 'Don Quixote',
+    author: 'Miguel de Cervantes',
+  }
+];
+
+console.log(library);
+
+var lastBook = library.pop()
+console.log('Value of lastBook',lastBook);
+
+var firstBook = library.shift();
+console.log('Value of firstBook',firstBook);
+
+var years = {
+  title: 'One Hundred Years of Solitude',
+  author: 'Gabriel Garcia Marquez',
+};
+
+var warPeace = {
+  title:'War and Peace',
+  author:'Leo Tolstoy',
+};
+
+library.push(years);
+library.unshift(warPeace);
+library.splice(1);
```

Comment:
there is an issue here. The final console log should show two books in the library, and you are currently only showing one!